guides/common/modules/proc_adding-azure-details-to-a-compute-profile.adoc

```diff
@@ -26,8 +25,16 @@ Note that the username that you enter for compute profile must be the same that
 . Optional: In the *Custom Script Command* field, enter commands to perform on the virtual machine when the virtual machine is provisioned.
 . Optional: If you want to run custom scripts when provisioning finishes, in the *Comma separated file URIs* field, enter comma-separated file URIs of scripts to use.
 The scripts must contain `sudo` at the beginning because {ProjectName} downloads files to the `/var/lib/waagent/custom-script/download/0/` directory on the host and scripts require sudo privileges to be executed.
+. If you want to add an *NVIDIA Driver* you can as well by selecting the *NVIDIA driver / CUDA* checkbox. 
+  For more information, refer to the following Microsoft Azure documentation:
+  - https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]
```

Comment:
```suggestion
** https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/hpccompute-gpu-linux[NVIDIA GPU Driver Extension for Linux]
```
Same for the other links.

Reply:
I don't understand the value of `**`?

Reply:
It indicates that the unordered list item is on a lower level than the ordered list:

. item 1
. item 2
** first bullet point for item2
** second bullet point for item2
. item 3

Reply:
But he's using an ordered list, with `.` so you'd need a first need a layer of an ordered list?

Reply:
Yes, going with `*` is also perfectly fine for asciidoc.

This is just for convencience, as it directly shows the "nested" aspect of the two lists.